src/ws/service/utils_handlers.rs

```diff
@@ -0,0 +1,216 @@
+use crate::{
```

Comment:
Instead of naming this file utils, let's separate concerns. For example: `mapper`, `friendship_status_calculation`, etc.